extensions/wikia/Chat2/Chat.class.php

```diff
@@ -290,6 +280,10 @@ protected static function getTimeLabel( $time ) {
 	 * @throws MWException
 	 */
 	public static function getBanInformation( $cityID, User $banUser ) {
+		if ( $banUser->isAnon() ) {
+			return false;
+		}
+
```

Comment:
Since ban rows in the DB require a user ID (and there no anonymous chatting) there's no reason to go any further if we've got an anonymous user).